kafka-rest/src/test/java/io/confluent/kafkarest/controllers/BrokerConfigManagerImplTest.java

```diff
@@ -37,14 +37,12 @@
 import javax.ws.rs.NotFoundException;
 import org.apache.kafka.clients.admin.Admin;
 import org.apache.kafka.clients.admin.AlterConfigOp;
-import org.apache.kafka.clients.admin.AlterConfigOp.OpType;
```

Comment:
Had added this extra import in the last PR, so removed it, and replaced its usage in the code to `AlterConfigOp.OpType`